src/emqx_protocol.erl

```diff
@@ -481,7 +505,14 @@ deliver({connack, ReasonCode}, PState) ->
 deliver({connack, ?RC_SUCCESS, SP}, PState = #pstate{zone = Zone,
                                                      proto_ver = ?MQTT_PROTO_V5,
                                                      client_id = ClientId,
+                                                     conn_props = CONNPROPS,
                                                      is_assigned = IsAssigned}) ->
+    ResponseInformation = case maps:find('Request-Response-Information', CONNPROPS) of
+                              {ok, 1} ->
+                                  iolist_to_binary(emqx_config:get_env(response_topic_prefix, "ResponseTopic"));
```

Comment:
this default value is not the same as in config. we should perhaps not allow a default value here ?